mne/io/meas_info.py

```diff
@@ -1723,7 +1723,7 @@ def create_info(ch_names, sfreq, ch_types=None, montage=None, verbose=None):
                          '(%s != %s)' % (len(ch_types), nchan))
     info = _empty_info(sfreq)
     info['meas_date'] = np.array([0, 0], np.int32)
-    loc = np.concatenate((np.zeros(3), np.eye(3).ravel())).astype(np.float32)
+    loc = np.zeros(12)
```

Comment:
is that a bug fix?

Reply:
@larsoner suggested to keep things simple and initialize with all zeroes (see his comment above).